src/audio/kpb.c

```diff
@@ -339,6 +339,11 @@ static int kpb_prepare(struct comp_dev *dev)
 
 	trace_kpb("kpb_prepare()");
 
+	if (kpb->state == KPB_STATE_RESETTING ||
+	    kpb->state == KPB_STATE_RESET_FINISH) {
```

Comment:
It's a little bit misleading. RESET_FINISH, but still we return EBUSY.

Reply:
I agree, what about `KPB_STATE_RESET_FINISHING` ?

Reply:
Sounds good.